buildenv/jenkins/common/build

```diff
@@ -178,15 +179,15 @@ def checkout_pullrequest() {
     }
 
     if (openjdk_bool) {
-        checkout_pullrequest(OPENJDK_PR)
+        checkout_pullrequest(OPENJDK_PR, "ibmruntimes/openj9-openjdk-jdk${SDK_VERSION}")
```

Comment:
This should fail now that we support JDK(next). I think SDK_VERSION is `next` but the repo is `ibmruntimes/openj9-openjdk-jdk`. Perhaps we need...
```
if (SDK_VERSION == "next") {
    CHECKOUT_REPO = "ibmruntimes/openj9-openjdk-jdk"
} else {
    CHECKOUT_REPO = "ibmruntimes/openj9-openjdk-jdk${SDK_VERSION}"
}
checkout_pullrequest(OPENJDK_PR, CHECKOUT_REPO)
```